azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -100,10 +104,17 @@ public Emailer(final Props props, final CommonMetrics commonMetrics) {
   }
 
   private void sendSlaAlertEmail(final SlaOption slaOption, final String slaMessage) {
-    final String subject = "SLA violation for " + getJobOrFlowName(slaOption) + " on " + getAzkabanName();
-    final String body = slaMessage;
+    final String subject =
+        "SLA violation for " + getJobOrFlowName(slaOption) + " on " + getAzkabanName();
     final List<String> emailList =
         (List<String>) slaOption.getInfo().get(SlaOption.INFO_EMAIL_LIST);
+    sendEmail(emailList, subject, slaMessage);
+  }
+
+  /**
+   * Send an email to the specified email list
+   */
+  public void sendEmail(final List<String> emailList, final String subject, final String body) {
```

Comment:
We already have azkaban.utils.EmailMessage#sendEmail method. Consider another method name to avoid confusion?

Reply:
it's fine since there're in different class and different argument. In addition, I couldn't think a better name than sendEmail :)